torch/_refs/__init__.py

```diff
@@ -5744,21 +5744,52 @@ def log_normal(self, mean=1, std=2, generator=None):
 
 
 # TODO: add support for functionalization aten.normal_functional
+# NOTE: we cannot use the out wrapper because of normal.float_float_out
```

Comment:
Why not?

Reply:
Right. In that case we need to implement the `out=` behaviour properly. At the moment we're not even copying the result into `out`